packages/nextjs/src/client/index.ts

```diff
@@ -38,7 +38,7 @@ const globalWithInjectedValues = global as typeof global & {
 export function init(options: BrowserOptions): void {
   applyTunnelRouteOption(options);
   buildMetadata(options, ['nextjs', 'react']);
-  options.environment = options.environment || process.env.NODE_ENV;
+  options.environment = options.environment || process.env.SENTRY_ENVIRONMENT || process.env.NODE_ENV;
```

Comment:
Wouldn't it be better to also add `|| VERCEL_ENV` before `|| process.env.NODE_ENV`? (per [vercel docs](https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables))

Reply:
And, given the platform restrictions on Node 16+, wouldn't `??` express the intent here better?